### ISSUE TITLE
MySQL fields not returned

### ISSUE THREAD

**Issue by brainsiq** [DataDog/dd-trace-js#1113]
**Describe the bug**

The `mysql` module's query function has three parameters in the callback as shown by the example in the README

https://github.com/mysqljs/mysql#introduction

> connection.query('SELECT 1 + 1 AS solution', function (error, results, fields) {
  if (error) throw error;
  console.log('The solution is: ', results[0].solution);
});

It appears that when using dd-trace-js that the final `fields` parameter is undefined.

**Environment**

* **Operation system:** Docker image `node:12.12.0-alpine`
* **Node version:** 12.12.0
* **Tracer version:** 0.26.1
* **Agent version:** 7.22.1
* **MySQL module version:** 2.14.1

**Comment by brainsiq** [comment]
I believe the bug is here: https://github.com/DataDog/dd-trace-js/blob/40e5617bb948bc0e14c71a4f57290242b0291a7b/packages/datadog-plugin-mysql/src/index.js#L59-L71

I assume this should have the callback parameters as the mysql library. I have tried to run use your tests to verify this but have not been able to get the tests to run because of issues getting the tests to run (firstly seemed I had to use node 8, and even then have errors from node-gyp when building the couchbase module).

**Comment by brainsiq** [comment]
In addition to this we can see an error in DataDog caused by calling `.forEach()` on the `fields` argument that refers to the LOC referenced above (/usr/src/app/node_modules/dd-trace/packages/datadog-plugin-mysql/src/index.js:70:5):

```
TypeError: Cannot read property 'forEach' of undefined
at /usr/src/app/lib/mysqlHelpers/index.js:540:22
at bound (domain.js:420:14)
at runBound (domain.js:433:12)
at Query.<anonymous> (/usr/src/app/node_modules/dd-trace/packages/datadog-plugin-mysql/src/index.js:70:5)
at /usr/src/app/node_modules/dd-trace/packages/dd-trace/src/scope/base.js:54:19
at Scope._activate (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/scope/async_hooks.js:53:14)
at Scope.activate (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/scope/base.js:12:19)
at Query.bound [as _callback] (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/scope/base.js:53:20)
at Query.Sequence.end (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Sequence.js:88:24)
at Query._handleFinalResultPacket (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Query.js:139:8)
at Query.EofPacket (/usr/src/app/node_modules/mysql/lib/protocol/sequences/Query.js:123:8)
at Protocol._parsePacket (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:279:23)
at Parser.write (/usr/src/app/node_modules/mysql/lib/protocol/Parser.js:76:12)
at Protocol.write (/usr/src/app/node_modules/mysql/lib/protocol/Protocol.js:39:16)
at Socket.<anonymous> (/usr/src/app/node_modules/mysql/lib/Connection.js:103:28)
at /usr/src/app/node_modules/dd-trace/packages/dd-trace/src/scope/base.js:54:19
```

**Comment by bengl** [comment]
@brainsiq This is fixed by #1116, and should be in the next release (0.27.0).
